plaidml/bridge/openvino/src/plaidml_plugin/plaidml_op_softmax.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <functional>
+#include <vector>
+
+#include "plaidml_op_softmax.hpp"
+#include "plaidml_util.hpp"
+
+#include "plaidml/op/op.h"
+
+void OpSoftmax::run(const plaidml::edsl::Tensor& I, plaidml::edsl::Tensor& O) {
+  auto* layer = dynamic_cast<SoftMaxLayer*>(layer_.get());
+  const auto& layout = layer_->outData.front()->getTensorDesc().getLayout();
+
+  std::vector<int64_t> to_nhwc = {0, 3, 1, 2};
```

Comment:
We'll need a fancier solution to channel ordering if OpenVINO ever uses more or fewer than 4 dims.